lib/common.sh

```diff
@@ -334,12 +334,6 @@ function init_minikube() {
     #If the vm exists, it has already been initialized
     if [[ "$(sudo virsh list --all)" != *"minikube"* ]]; then
       sudo su -l -c "minikube start --insecure-registry 192.168.111.1:5000" "$USER"
-      # Pre-pull the image to reduce pod initialization time
-      for IMAGE_VAR in IRONIC_IMAGE IPA_DOWNLOADER_IMAGE IRONIC_INSPECTOR_IMAGE BAREMETAL_OPERATOR_IMAGE; do
-        IMAGE=${!IMAGE_VAR}
-        sudo su -l -c "minikube ssh sudo docker pull $IMAGE" "${USER}"
-      done
-      sudo su -l -c "minikube ssh sudo docker image ls" "${USER}"
```

Comment:
This is for minikube, was this also done twice? or minikube will fetch the local images?

Reply:
This was done both in metal3-dev-env and in minikube, both fetching the image from quay.io